backend/eventloop/db_reconciler.go

```diff
@@ -190,43 +191,47 @@ func cleanOrphanedEntriesfromTable_DTAM_DeleteEntry(ctx context.Context, deplToA
 		}
 	}
 
-	log := logger.WithValues("applicationID", deplToAppMapping.Application_id)
-
 	// 1) Remove the ApplicationState from the database
-	if err := deleteDbEntry(ctx, dbQueries, deplToAppMapping.Application_id, dbType_ApplicationState, log, deplToAppMapping); err != nil {
+	if err := deleteDbEntry(ctx, dbQueries, deplToAppMapping.Application_id, dbType_ApplicationState, logger, deplToAppMapping); err != nil {
 		return err
 	}
 
+	logger.Info("Managed-gitops clean up job for DB deleted ApplicationState: " + deplToAppMapping.Application_id + " entry from DB.")
+
 	// 2) Set the application field of SyncOperations to nil, for all SyncOperations that point to this Application
 	// - this ensures that the foreign key constraint of SyncOperation doesn't prevent us from deletion the Application
 	rowsUpdated, err := dbQueries.UpdateSyncOperationRemoveApplicationField(ctx, deplToAppMapping.Application_id)
 	if err != nil {
-		log.Error(err, "unable to update old sync operations", "applicationId", deplToAppMapping.Application_id)
+		logger.Error(err, "unable to update old sync operations", "applicationId", deplToAppMapping.Application_id)
 		return err
 	} else if rowsUpdated == 0 {
-		log.Info("no SyncOperation rows updated, for updating old syncoperations on GitOpsDeployment deletion")
+		logger.Info("no SyncOperation rows updated, for updating old syncoperations on GitOpsDeployment deletion")
 	} else {
-		log.Info("Removed references to Application from all SyncOperations that reference it")
+		logger.Info("Removed references to Application from all SyncOperations that reference it")
 	}
 
 	// 3) Delete DeplToAppMapping row that points to this Application
-	if err := deleteDbEntry(ctx, dbQueries, deplToAppMapping.Deploymenttoapplicationmapping_uid_id, dbType_DeploymentToApplicationMapping, log, deplToAppMapping); err != nil {
+	if err := deleteDbEntry(ctx, dbQueries, deplToAppMapping.Deploymenttoapplicationmapping_uid_id, dbType_DeploymentToApplicationMapping, logger, deplToAppMapping); err != nil {
 		return err
 	}
 
+	logger.Info("Managed-gitops clean up job for DB deleted DeploymentToApplicationMapping: " + deplToAppMapping.Deploymenttoapplicationmapping_uid_id + " entry from DB.")
```

Comment:
For example, I assume this (and others) could be instead logged in `deleteDbEntry`.